scripts/kconfig/kconfigfunctions.py

```diff
@@ -228,6 +228,32 @@ def _node_array_prop(node, prop, index=0, unit=None):
     or string hex value. If the property 'prop' is not found or the given 'index'
     is out of range it will return 0.
 
+    The function will divide the value based on 'unit':
+        None        No division
+        'k' or 'K'  divide by 1024 (1 << 10)
+        'm' or 'M'  divide by 1,048,576 (1 << 20)
+        'g' or 'G'  divide by 1,073,741,824 (1 << 30)
+    """
+    if not node:
+        return 1
+
+    if prop not in node.props:
+        return 2
+    if node.props[prop].type != "array":
+        return 3
+    if int(index) >= len(node.props[prop].val):
+        return 4
```

Comment:
Should the function really return these values? Or is this a leftover from when you tested your changes?

Reply:
Good catch. Leftover indeed